src/types.h

```diff
@@ -78,6 +78,12 @@ enum class StackMode : uint8_t
   raw,
 };
 
+const std::map<StackMode, std::string> STACK_MODE_NAME_MAP = {
```

Comment:
types used to have this map reversed but I moved it to config.h:
https://github.com/iovisor/bpftrace/blob/master/src/config.h#L71

If you want (or I can do it it another PR), just keep the one here and in the Config constructor, build the reverse of this to be used in config.cpp

Reply:
Yeah, I noticed, but this one doesn't logically belong to `config.h`. Building the reverse one dynamically probably makes sense so that we don't forget to update both if we add new stack types. Let me add that to this PR.

Reply:
Done.

Reply:
Thanks!